### PR TITLE
Finished Home page

### PR DESCRIPTION
**Pull request by Chimoha31**
I finished coding of home page for both, mobile and laptop.

### CODE REVIEW

File: Santouka/css/main.css [hunk 1]
```diff
@@ -69,3 +69,164 @@ header
...
+  font-weight: bolder;
+}
+
+.main_container strong {
+  color: rgb(185, 8, 8);
+  font-size: 2rem;
+}
+
+
+.fadein:nth-child(1) {
```
Comment (SOunit) [inline Santouka/css/main.css]: let's move same properties to common style!

.fadein {
opacity: 0;
  animation: fadein 6s ease forwards;
}

.fadein:nth-child(1){
animation-delay: 1s;
}

.fadein:nth-child(2){
animation-delay: 4s;
}

.fadein:nth-child(3){
animation-delay: 7s;
}

File: Santouka/css/main.css [hunk 2]
```diff
@@ -69,3 +69,164 @@ header
...
+  align-items: center;
+  justify-content: center;
+}
+
+.btn a {
+  display: block;
+  color: black;
+}
+
+.btn btn {
```
Comment (SOunit) [inline Santouka/css/main.css]: .btn-wrapper .btn

is better i guess

Comment (SOunit) [reply Santouka/css/main.css]: from the name .btn, i expect this style is for button itself!

but from .btn btn, i guess .btn is actually a button wrapper.
in that case, if you use 

.btn-wrapper .btn

then i can clearly understand the actual contents, html structure from css-style name!

**Review by SOunit — CHANGES_REQUESTED** [review]
please check comments!

i just leave you if change or not.

File: Santouka/css/main.css (hunk 2 above)
Comment (Chimoha31) [reply Santouka/css/main.css]: I fixed it!

File: Santouka/css/main.css (hunk 1 above)
Comment (Chimoha31) [reply Santouka/css/main.css]: I fixed it !

Comment (SOunit) [reply Santouka/css/main.css]: thank you!

button become button_container, but

.fadein still has same values....

File: Santouka/js/menu.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
+console.log('hi from js');
+
+// {
```
Comment (SOunit) [inline Santouka/js/menu.js]: you need comment about comment out src!

File: Santouka/menu.html [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+  <link
+    href="https://fonts.googleapis.com/css2?family=Urbanist:wght@300;700&display=swap"
+    rel="stylesheet"/>
+  <link
+    href="https://fonts.googleapis.com/icon?family=Material+Icons"
+    rel="stylesheet"/>
+  <link rel="stylesheet" href="css/menu.css">
+</head>
+
+<!---------------------
```
Comment (SOunit) [inline Santouka/menu.html:18]: why are they comment out?

File: Santouka/css/menu.css [hunk 5]
```diff
@@ -0,0 +1,313 @@
+* {
```
Comment (SOunit) [inline Santouka/css/menu.css]: same property in shared css, so you dont need this

**Review by SOunit — CHANGES_REQUESTED** [review]
please delete .DS_Store files!
and i added some comments too!

File: Santouka/css/main.css (hunk 1 above)
Comment (Chimoha31) [reply Santouka/css/main.css]: I fixed it !

File: Santouka/menu.html (hunk 4 above)
Comment (Chimoha31) [reply Santouka/menu.html:18]: I tried to combine Ayaka's JS, but doesn't work yet. So I just commented Ayaka's JS out until I combine it to my page!

File: Santouka/css/menu.css (hunk 5 above)
Comment (Chimoha31) [reply Santouka/css/menu.css]: I deleted it!

**Review by SOunit — COMMENTED** [review]
lgtm

**Pull request merged**